invenio/base/wrappers.py

```diff
@@ -92,7 +96,7 @@ def decorator(*args, **kwargs):
         url = wrapped(*args, **kwargs)
```

Comment:
as `url_scheme_prefixes` should be already in `unicode` we should *unicodify* only the result of `wrapped` function.